cmake/configs/posix_sdflight_default.cmake

```diff
@@ -78,8 +81,17 @@ set(config_module_list
 	lib/tailsitter_recovery
 	lib/version
 	lib/DriverFramework/framework
+	lib/micro-CDR
 
 	platforms/common
 	platforms/posix/px4_layer
 	platforms/posix/work_queue
 	)
+
+set(config_rtps_send_topics
```

Comment:
I don't know yet, but I see several reasons why we should have a single list, at least in the beginning:
- If someone tests & implements something on one platform, others on different platforms fail to reproduce it simply because the topic list is different. It's a source for errors that hinders development.
- a single list brings more awareness which topics (or what information) are communicated accross FMU boundary. Some topics are clearly only for internal use. It can be fatal if someone just publishes arbitrary topics into the FMU. It can lead to very hard-to-debug async bugs if we don't keep an eye on the set of topics.